### DISCUSSION TITLE
Carousel in Product recommendation

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#267] [1. Help] [answered]
Hi @vovayatsyuk 

Is there any way to add a carousel in the HTML template, I am trying to add it into product recommendation HTML template with the below code:
``` 
data-mage-init='{"Magento_PageBuilder/js/content-type/slider/appearance/default/widget":{"arrows":true,"dots":true}}'
 ```
but it's not working.

**Comment by vovayatsyuk** [comment]
How do you add it exactly? Template path, which line?

p.s. Here are other possible solutions: https://github.com/orgs/breezefront/discussions/185#discussioncomment-6768272

**Comment by pk-pankaj** [comment]
Hi @vovayatsyuk

I tried adding this to this template
vendor/swissup/module-breeze-magento-enterprise/view/frontend/web/template/product-recommendations/recommendations.html 
on line number 4 with "block-content" and line 5 with "products-grid" and line 16 with "product-items" 
but did not work at any line.

It's working when trying with "breeze-carousel" class but how can I pass carousel options with it like dots and arrows.

**Reply by vovayatsyuk** [reply, marked as answer]
Here is how we handle “breeze-carousel” class name: https://github.com/breezefront/theme-frontend-breeze-blank/blob/dc2495100fe80b9725aa0d758ec4216b3f5e3ea6/web/js/breeze/theme-blank.js#L70

You can use similar code for another class name to create slider with required options.
